app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -5,6 +5,10 @@ type Query {
     customer: Customer @resolver(class: "Magento\\CustomerGraphQl\\Model\\Resolver\\Customer") @doc(description: "The customer query returns information about a customer account")
 }
 
+type Mutation {
+    changePassword(currentPassword: String!, newPassword: String!): Customer @resolver(class: "\\Magento\\CustomerGraphQl\\Model\\Resolver\\Customer\\Account\\ChangePassword") @doc(description:"Changes password for logged in customer")
```

Comment:
Since this mutation is in global scope, it is better to name it `changeCustomerPassword`